contributing-workflow.md

```diff
@@ -80,8 +90,26 @@ The package naming conventions follow SemVer 2.0.0 and also provide changeset in
 Package names will be like: `Microsoft.Data.SqlClient.1.1.0-build.19250.1-c21aa7c.nupkg`
 
 Breaking down:
-- `1.1.0-build` - Identitier for currently active driver version in Build.
+- `1.1.0-build` - Identifier for currently active driver version in Build.
 - `19250.1` - Unique identifier to keep latest PRs on top of the feed.
 - `c21aa7c` - Short Commit Id to identify merged commit in `master`.
 
 > Note: This public feed is only for testing and validating new PR changes. Packages from feed will be eventually removed when the maximum NuGet Package limit of **50** is reached. We do not recommend using packages from this feed in client applications.
+
+## Nightly builds
+
+We also maintain NuGet feed for Nightly builds that are generated when new changes are merged in the repository. To access Nightly builds, add below configuration to package sources:
```

Comment:
NIT: should we change it to `nightly build` with lower case ?

Reply:
Headings are often sentence case.